app/mailers/concerns/multilingual_mailer.rb

```diff
@@ -7,14 +7,18 @@
 
 module MultilingualMailer
   extend ActiveSupport::Concern
+  include ActionView::Helpers::TagHelper
 
-  LANGUAGE_SEPARATOR = "<br><br>--------------------<br><br>"
   SANITIZE_TAG_WHITELIST = %w[a br div]
 
   delegate :sanitize, to: "ActionController::Base.helpers"
```

Comment:
Diese Änderung, welche da bereits vor ein paar Tagen rein kam (07a6c2a7c6a74a43070b416b20e78800e3c12f67), passt nicht in das neue Konzept rein. Insbesondere soll hier nicht anders verfahren werden als in den anderen Mailers. Also kein sanitize, sondern escaping. Kannst du das bitte ebenfalls noch anpassen und auch gleich Specs ergänzen, damit das auch für den MultilingualMailer gut abgedeckt ist.

Allenfalls könnte ein initiales Sanitizing vom CustomContent Body Sinn machen (im Core), in dem Fall müssten aber alle Tags und Attribute, welche via Trix erstellt werden können, dort auch weiterhin erlaubt sein.